datasets/rdw/rdw.json

```diff
@@ -67,28 +75,41 @@
             "type": "string",
             "description": "Type voertuig."
           },
-          "aantalAssen": {
-            "type": "integer",
-            "description": "Aantal assen aanwezig."
+          "assen": {
```

Comment:
Hier zat ik vanmiddag de verkeerde kant op te denken. Doordat het een array wordt, heb je natuurlijk al flexibiliteit in het aantal assen!